code/game/objects/structures/therapy.dm

```diff
@@ -0,0 +1,586 @@
+				return
+
+			text = input("What would you like to suggest?", "Hypnotic suggestion", null, null)
+			text = sanitize(text)
+			if(!text)
+				return
+
+			var/thrall_response = alert(thrall, "Do you believe in hypnosis?", "Willpower", "Yes", "No")
+			if(thrall_response == "Yes")
+				thrall << "<span class='notice'><i>... [text] ...</i></span>"
+				thrall.cure_all_traumas(cure_type = CURE_HYPNOSIS)
+			else
+				thrall = null
+
+/obj/item/weapon/mesmetron/afterattack(obj/target, mob/user, proximity)
+	if(!proximity)
+		return
+
+	if(!ishuman(target))
+		return
+
+	var/mob/living/carbon/human/L = target
+
+	user.visible_message("<span class='warning'>[user] begins to mesmerizingly wave [src] like a pendulum before [L]'s very eyes!</span>")
+
+	if(!do_mob(user, L, 10 SECONDS))
+		return
+
+
+	if((!user in view(1,target)))
+		return
+
+	var/response = alert(L, "Do you believe in hypnosis?", "Willpower", "Yes", "No")
+
+	if(response == "Yes")
+		L.visible_message("<span class='warning'>[L] falls into a deep slumber!</span>", "<span class ='danger'>You fall into a deep slumber!</span>")
+
+		L.sleeping = max(L.sleeping, 40)
+		L.drowsyness = max(L.drowsyness, 60)
+		thrall = L
+		START_PROCESSING(SSfast_process, src)
+
+/obj/structure/metronome
+	name = "metronome"
+	desc = "Tick. Tock. Tick. Tock. Tick. Tock."
+	icon = 'icons/obj/structures.dmi'
+	icon_state = "metronome1"
+	anchored = 1
+	density = 0
+	var/ticktock = "Tick"
+
+/obj/structure/metronome/Initialize()
+	. = ..()
+	START_PROCESSING(SSslow_process, src)
+
+/obj/structure/metronome/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(iswrench(W))
+		playsound(src.loc, 'sound/items/Ratchet.ogg', 50, 1)
+		if(anchored)
+			user << "<span class='notice'>You unanchor \the [src] and it destabilizes.</span>"
+			STOP_PROCESSING(SSslow_process, src)
+			icon_state = "metronome0"
+			anchored = 0
+		else
+			user << "<span class='notice'>You anchor \the [src] and it restabilizes.</span>"
+			START_PROCESSING(SSslow_process, src)
+			icon_state = "metronome1"
+			anchored = 1
+	else
+		..()
+
+/obj/structure/metronome/process()
+	var/counter = 0
+	var/mob/living/carbon/human/H
+	for(var/mob/living/L in view(3,src.loc))
+		counter++
+		if(ishuman(L))
+			H = L
+	if(counter == 1 && H)
+		if(ticktock == "Tick")
+			ticktock = "Tock"
+		else
+			ticktock = "Tick"
+		H << "<span class='notice'><i>[ticktock]. . .</i></span>"
+		H << 'sound/effects/singlebeat.ogg'
+		if(prob(1))
+			H.cure_all_traumas(cure_type = CURE_SOLITUDE)
+
+/obj/machinery/chakrapod
+	name = "Crystal Therapy Pod"
+	desc = "A state-of-the-art crystal therapy pod. Designed to utilize phoron enhanced quartz crystals to remove mental trauma from the body. Proven to be 100% effective 30% of the time!"
+	icon = 'icons/obj/Cryogenic2.dmi'
+	icon_state = "syndipod_0"
+	density = 1
+	anchored = 1
+
+	use_power = 1
+	idle_power_usage = 60
+	active_power_usage = 10000
+
+	var/mob/living/carbon/occupant
```

Comment:
Same deal here. The human cannot effectively clean itself up from here, so this should be a weakref.